src/main/java/com/avairebot/commands/music/RepeatMusicQueueCommand.java

```diff
@@ -88,14 +91,25 @@ public boolean onCommand(CommandMessage context, String[] args) {
         if (!musicManager.canPreformSpecialAction(this, context, "loop queue")) {
             return false;
         }
-
-        musicManager.setRepeatQueue(!musicManager.isRepeatQueue());
-
-        context.makeSuccess(context.i18n("success"))
-            .set("status", musicManager.isRepeatQueue()
-                ? context.i18n("enabled") : context.i18n("disabled"))
-            .queue(message -> message.delete().queueAfter(5, TimeUnit.MINUTES, null, RestActionUtil.ignore));
-
-        return true;
+        if (args[0].equals("single")) {
```

Comment:
Using `.equalsIgnoreCase` instead of `.equals` here would allow people to use the command correctly more often if they hit shift by mistake or have caps lock turned on, another option is using a switch statement on the first argument that is converted to all lower case, that way you can do a lot of checks very quickly, something like:
```java
switch (args[0].toLowerCase()) {
    case "a":
    case "al":
    case "all":
        // Do stuff related to all
        break;

    case "one":
    case "single":
        // Do stuff related to single
        break;
        
        // Etc..
}
```
You could easily add or remove triggers for the different types that way, would also be more readable than having 10 if and if-else statements right after each other.